### ISSUE TITLE
NTR: Loss of primary podocyte processes (level 4)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6341]
Loss of primary podocyte processes (level 4)
parent:Abnormalities of the visceral epithelial cells ()
def:podocyte cell body sitting directly on underlying GBM, usually with extensive foot process effacement (Barisoni et al) @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh
Can we define how this is different from *Podocyte foot process effacement*? Given that effacement is part of the definition of the new term, when should we use the new term, and when should we use *Podocyte foot process effacement*

https://hpo.jax.org/app/browse/term/HP:0031266

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh -- should we add this as a synonym to Podocyte foot process effacement?

**Comment by avizrosenberg** [comment]
yes this should be a synonym

**Comment by yongqunh** [comment]
Yes. I agree.

**Comment by pnrobinson** [comment]
done, thank you!
